news/changelog.md.tt

```diff
@@ -0,0 +1,18 @@
+# Changelog
+
+When a release is created, that branch is forked off, and its changelog
+is also forked. For example, none of the changes after 0.9.8n appear in
+the other logs, because 1.0.0 was created after that release and before
+0.9.8o. Any changes that are merged across branches, however, should
```

Comment:
Is it worth updating the version numbers used for the example to something slightly more current?